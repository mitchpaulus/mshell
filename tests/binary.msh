`binary.txt` readFileBytes bytes!
@bytes str wl
@bytes utf8Str w
