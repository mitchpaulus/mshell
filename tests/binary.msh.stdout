48656c6c6f20c2b0460a
Hello Â°F
